server/routes/disasters.py

```diff
@@ -66,6 +67,12 @@ def disaster_dashboard(place_dcid=None):
     dashboard_config.metadata.contained_place_types.update(
         place_metadata.contained_place_types_override)
 
+  child_places = place_api.child_fetch(place_dcid)
```

Comment:
we could move this to a helper so it's returned in lib_subject_page_config.place_metadata (or I can do it, if this goes in first).

Reply:
since I think this will be going in first, I added a TODO for you to move this to a helper